x-pack/plugins/triggers_actions_ui/public/plugin.ts

```diff
@@ -31,6 +32,8 @@ interface PluginsStart {
   data: DataPublicPluginStart;
   charts: ChartsPluginStart;
   management: ManagementStart;
+  alerting: AlertingStart;
```

Comment:
To handle change to optional plugin:
```suggestion
  alerting?: AlertingStart;
```